wwwroot/test/WMS/multiple_style_legend_url.xml

```diff
@@ -102,7 +102,7 @@
                         <Format>image/gif</Format>
                         <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink"
                                         xlink:type="simple"
-                                        xlink:href="http://www.example.com/legendUrl"/>
+                                        xlink:href="http://www.example.com/request=GetLegendGraphic?firstUrl"/>
```

Comment:
This isn't actually a valid `GetLegendGraphic` URL.  It should be:
http://www.example.com?request=GetLegendGraphic&firstUrl